Vendor/Networking/NetworkActivityIndicator.swift

```diff
@@ -36,7 +36,9 @@ class NetworkActivityIndicator: NSObject {
             #if os(iOS)
                 let deadline = DispatchTime.now() + Double(Int64(0.5 * Double(NSEC_PER_SEC))) / Double(NSEC_PER_SEC)
                 DispatchQueue.main.asyncAfter(deadline: deadline) {
-                    UIApplication.shared.isNetworkActivityIndicatorVisible = (self.activitiesCount > 0)
+                    if UIApplication.responds(to: NSSelectorFromString("shared")) {
```

Comment:
You can't access `shared` from inside an app extension. I thought either this or some `#ifdef`...